Mathlib/Data/Set/Image.lean

```diff
@@ -421,6 +421,9 @@ theorem image_preimage_subset (f : α → β) (s : Set β) : f '' (f ⁻¹' s) 
 theorem subset_preimage_image (f : α → β) (s : Set α) : s ⊆ f ⁻¹' (f '' s) := fun _ =>
   mem_image_of_mem f
 
+theorem preimage_image_univ {f : α → β} : f ⁻¹' (f '' univ) = univ :=
```

Comment:
Presumably `image_preimage_empty` is also true, to match the `bot` lemma we wrote for the GaloisConnection case.